src/frontend/web_application/src/modules/draftMessage/components/DraftMessage/presenter.jsx

```diff
@@ -647,11 +647,8 @@ class DraftMessage extends Component {
             disabled={!canSend}
           >
             {this.state.isSending && (<Spinner display="inline" theme="bright" />)}
-            {!this.state.isSending && (<Icon type="laptop" />)}
-            {' '}
-            ---
-            {' '}
-            <Icon type="user" />
+            {this.state.isSending && ' '}
```

Comment:
`!this.state.isSending && …`

Reply:
This space is not meant as a placeholder for spinner but to actually add a space between spinner and icon.

Could not add it to the first condition, eslint complains.

Reply:
oups I didn't read correctly

what about:
```
{this.state.isSending && (<Spinner display="inline" theme="bright" />)}
{!this.state.isSending && (<Icon type="paper-plane" />)}
```

The idea is to to not change the size of the button